src/styles/product.css

```diff
@@ -0,0 +1,73 @@
+.product {
+  border: 1px solid #f3f3f3;
+  width: 166px;
```

Comment:
`card width is 200px including border`
check the task description again

Reply:
+32px paddings (16+16)

Reply:
I would suggest using `box-sizing` property to address this issue